src/main/java/com/alipay/oceanbase/rpc/ObTableClient.java

```diff
@@ -1560,14 +1559,15 @@ public ObPair<Long, ObTableParam> getTable(String tableName, Object[] rowKey, bo
      * @param rowKey row key
      * @param refresh whether to refresh
      * @param waitForRefresh whether wait for refresh
+     * @param needFetchAll whether to fetch all
      * @param route ObServer route
      * @return ObPair of partId and table
      * @throws Exception exception
      */
     public ObPair<Long, ObTableParam> getTable(String tableName, Object[] rowKey, boolean refresh,
```

Comment:
needFetchAll should be the last param and it should has default value

Reply:
Already refactor getTable interfaces. Only keep the getTable interface already used by users. The needFetchAll has been set to false for the user interface.